language/de-DE/joomla.ini

```
@@ -196,7 +196,7 @@ JFIELD_MEDIA_EMBED_CHECK_DESC_LABEL="Native Elemente audio, video oder object ve
 JFIELD_MEDIA_EMBED_CHECK_LABEL="Einbetten"
 JFIELD_MEDIA_CLASS_LABEL="CSS-Klasse Bild"
 JFIELD_MEDIA_FIGURE_CAPTION_LABEL="Bildunterschrift"
-JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse Bildunterschrift"
+JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse Bild-Container"
```

```suggestion
JFIELD_MEDIA_FIGURE_CLASS_LABEL="CSS-Klasse figure-Element"
```

Das trifft es doch ganz gut. 
Container ist mir zu weit weg. 

https://wiki.selfhtml.org/wiki/HTML/Elemente/figure

---

CSS Klasse Medien-Element? figure-Element ist auch gut. Ich schau heute Nachmittag nochmal rein, dann machen wir das heute fix

---

Da es sich direkt auf das HTML-figure bezieht, bringt Medien-Element wieder eine Unschärfe rein.

---

Ich habe die beiden editiert, ich hoffe im richtigen Branch. Benötigen die einen erneuten Pull? Wie gesagt, ich muss mich noch auf github einfinden, und bräuchte eventuell eure Unterstützung. Danke. 


> https://github.com/joomlagerman/joomla/blob/b2ee0030ea0490cf5e23390a108efdb24516bc7c/administrator/language/de-DE/joomla.ini#L253
> 
> 
> https://github.com/joomlagerman/joomla/blob/b2ee0030ea0490cf5e23390a108efdb24516bc7c/api/language/de-DE/joomla.ini#L252

---

Ich hatte doch alle schon auf figure-Element geändert :)